### ISSUE TITLE
1.9.2 Ping timeout on connect

### ISSUE THREAD

**Issue by tsolace** [PrismarineJS/mineflayer#417]
Player connects and shows up, but then times out (the server says so)
```
AssertionError: Deserialization error for play.toClient : 86272 === 43921
    at addColumn (/home/white/Programs/minebots/minebot/node_modules/mineflayer/lib/plugins/blocks.js:85:12)
    at Client.<anonymous> (/home/white/Programs/minebots/minebot/node_modules/mineflayer/lib/plugins/blocks.js:188:5)
    at emitTwo (events.js:87:13)
    at Client.emit (events.js:172:7)
    at Parser.<anonymous> (/home/white/Programs/minebots/minebot/node_modules/mineflayer/node_modules/minecraft-protocol/dist/client.js:101:16)
    at emitOne (events.js:77:13)
    at Parser.emit (events.js:169:7)
    at readableAddChunk (/home/white/Programs/minebots/minebot/node_modules/mineflayer/node_modules/minecraft-protocol/node_modules/readable-stream/lib/_stream_readable.js:198:18)
    at Parser.Readable.push (/home/white/Programs/minebots/minebot/node_modules/mineflayer/node_modules/minecraft-protocol/node_modules/readable-stream/lib/_stream_readable.js:157:10)
    at Parser.Transform.push (/home/white/Programs/minebots/minebot/node_modules/mineflayer/node_modules/minecraft-protocol/node_modules/readable-stream/lib/_stream_transform.js:123:32)
```

**Comment by rom1504** [comment]
1.9 is not supported yet #1.

**Comment by hldgaofeng** [comment]
When to support 1.9? Do you have a plan?

**Comment by rom1504** [comment]
Yeah the plan is getting this done https://github.com/PrismarineJS/prismarine-chunk/issues/19 then we can use it to parse the chunk and get the rest of mineflayer updated (I want to try keeping 1.8 compatibility in the process)

**Comment by rom1504** [comment]
closed in favor of #449
